crates/packager/src/package/app/mod.rs

```diff
@@ -52,6 +52,65 @@ pub(crate) fn package(ctx: &Context) -> crate::Result<Vec<PathBuf>> {
 
     tracing::debug!("Copying frameworks");
     let framework_paths = copy_frameworks_to_bundle(&contents_directory, config)?;
+
+    // All dylib files and native executables should be signed manually
+    // It is highly discouraged by Apple to use the --deep codesign parameter in larger projects.
+    // https://developer.apple.com/forums/thread/129980
+    for framework_path in &framework_paths {
+        if let Some(framework_path) = framework_path.to_str() {
+            // Find all files in the current framework folder
+            let constructed_glob = format!("{}/**/*", framework_path);
```

Comment:
isn't it better to use [walkdir](https://docs.rs/walkdir/latest/walkdir/)?

Reply:
Ah yea, I didn't see that you already have a dependency on walkdir and didn't want to add one. I'll change that